pkg/logging/config_test.go

```diff
@@ -140,7 +140,7 @@ func TestNewConfig(t *testing.T) {
 	c, err := NewConfigFromConfigMap(&corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: system.Namespace(),
-			Name:      "config-logging",
+			Name:      "serving-config-logging",
```

Comment:
Why change the default name? It should stay the same (i.e. config-logging). The comment is applicable to all the lines here that changes the default name,

Reply:
I change the default name back to `config-logging`